config.go

```diff
@@ -16,20 +16,19 @@ type Config struct {
 	// the final level.
 	ContributionsPerc int
 
-	// LevelTimeout is used to decide when a Handel nodes passes to the next
-	// level even if it did not receive enough signatures. If not specified, a
-	// timeout of 100ms is used by default.
-	LevelTimeout time.Duration
+	// UpdatePeriod indicates at which frequency a Handel nodes sends updates
+	// about its state to other Handel nodes.
+	UpdatePeriod time.Duration
+
+	// UpdateCount indicates the number of nodes contacted during each update at
+	// a given level.
+	UpdateCount int
 
 	// CandidateCount indicates how many peers should we contact each time we
```

Comment:
I already wrote it in another comment in this review, but I think that CandidateCount doesn't really match what handel is doing; using 'nodes' instead of 'candidates' would be better.